games-vue-client/src/components/lobby/LobbyGameType.vue

```diff
@@ -61,7 +61,7 @@ export default {
             return supportedGames.games[this.gameType];
         },
         displayName() {
-            return this.supportedGame.displayName ? this.supportedGame.displayName : this.gameType;
+            return this.supportedGame?.displayName || this.gameType;
```

Comment:
Oh, does `?.` work in JavaScript ? Cool! Although I think `this.supportedGame` is never null (or is it?)

Reply:
Yeah, it's a nice one with `?.` 

For me, this.supportedGame was null but perhaps it was only and when running against Zomis server.